R/escape.R

```diff
@@ -273,4 +293,6 @@ sql_escape_logical.DBIConnection <- function(con, x) {
   y
 }
 
-
+sql_escape_raw.DBIConnection <- function(con, x) {
```

Comment:
This needs to be exported

Reply:
Can you also add a comment with link to some docs about this sql feature?

Reply:
Added @export and comments